datalake/src/main/java/com/sequenceiq/datalake/service/validation/diagnostics/DiagnosticsCollectionValidator.java

```diff
@@ -50,6 +52,16 @@ public void validate(DiagnosticsDestination destination, StackV4Response stackV4
         }
     }
 
+    private void validateStackStatus(StackV4Response stackV4Response) {
+        Status stackStatus = stackV4Response.getStatus();
+        if (Status.STOPPED.equals(stackStatus)
+                || Status.DELETED_ON_PROVIDER_SIDE.equals(stackStatus)
+                || Status.REQUESTED.equals(stackStatus)) {
+            throw new BadRequestException(String.format("Cannot run diagnostics on stack (crn: '%s') if the stack is in %s state",
```

Comment:
I think the crn is not useful for the enduser also do not call it stack but use CDP terms like Data Hub or Data Lake

this class seems to me a duplication.